archiso/airootfs/root/.automated_script.sh

```diff
@@ -31,6 +31,4 @@ automated_script ()
 
 if [[ $(tty) == "/dev/tty1" ]]; then
     automated_script
-    echo "After running pacstrap and before you go into chroot, please, *please* copy /t2kern to /mnt/t2kern just in case."
```

Comment:
Add these lines again (and t2kern.txt), without these you can't access the local repo